### PR TITLE
Setup sync target action

### PR DESCRIPTION
**Pull request by igorbrasileiro**
## What's the purpose of this pull request?
The Template Repository feature is a great way to accelerate the creation of new projects. However, once a repository is created it won't receive any changes/updates done on the template repository.

So, this PR adds a GitHub action that syncs storeframework.store with this template. 

## How it works? 
This new GitHub action is triggered when the push event happens on the branch `master` and adds all files on the `storeframework.store` repo.

## How to test it?
https://github.com/vtex-sites/base.store/runs/4884237489?check_suite_focus=true

## References
<em>Spread the knowledge: is this any content you used to create this PR that is worth sharing?</em>  

<em>Extra tip: add references to related issues or mention people important to this PR may be good for the documentation and reviewing process</em>

### CODE REVIEW

File: .github/template-sync.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# NOTE There is currently a bug in the GitHub APIs preventing this action from automatically detecting dependent repositories, until this is tis resolved, please use additional property in the config file to manually include repositories you want to sync
+additional:
+  - "storeframework.store"
+
+files:
+  - "!package.json"
+  - "!yarn.lock"
+   # you probably want to exclude these files:
```
Comment (lariciamota) [inline .github/template-sync.yml:8]:
```suggestion
```

File: .github/template-sync.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# NOTE There is currently a bug in the GitHub APIs preventing this action from automatically detecting dependent repositories, until this is tis resolved, please use additional property in the config file to manually include repositories you want to sync
```
Comment (lariciamota) [inline .github/template-sync.yml:1]:
```suggestion
# NOTE There is currently a bug in the GitHub APIs preventing this action from automatically detecting dependent repositories, until this is resolved, please use additional property in the config file to manually include repositories you want to sync
```

File: .github/workflows/template-sync.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+  template_sync:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2 # important!
+      - uses: ahmadnassri/action-template-repository-sync@v1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          dry-run: true
```
Comment (lariciamota) [inline .github/workflows/template-sync.yml:18]: Do you want to merge with dry-run set to true?

**Review by lariciamota — COMMENTED** [review]
Nice! 
Did you test it somehow?

File: .github/template-sync.yml (hunk 1 above)
Comment (igorbrasileiro) [reply .github/template-sync.yml:8]: > Nice! Did you test it somehow?

Take a look at actions on this PR.

File: .github/workflows/template-sync.yml (hunk 3 above)
Comment (igorbrasileiro) [reply .github/workflows/template-sync.yml:18]: Yeah, just for testing before merge it. I will also remove this branch from triggers branches.

**Pull request closed without merging**